homeassistant/const.py

```diff
@@ -456,6 +456,8 @@ class Platform(StrEnum):
 # Temperature attribute
 ATTR_TEMPERATURE: Final = "temperature"
 
+# Persons attribute
+ATTR_PERSONS: Final = "persons"
```

Comment:
persons or people? 

> For references to groups of a specific or general number, either people or persons may be used, but modern style guides tend to prefer people where earlier guides preferred persons, especially for countable groups. 

https://www.merriam-webster.com/words-at-play/people-vs-persons

Reply:
It are `person` entities... ? So, besides grammar, I'm unsure. It is referring to entities, not actual human beings.

🤷 I don't have a strong opinion, what do you think?

Reply:
I think `persons` is better as we're referring to a specific set of people:
https://www.merriam-webster.com/words-at-play/people-vs-persons

Reply:
let's do persons.